full-v3.yml

```diff
@@ -0,0 +1,2299 @@
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/ContactsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/mystudents:
+    get:
+      tags: ["Users"]
+      description: Returns the students for a user
+      operationId: getStudentsForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/StudentsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/myteachers:
+    get:
+      tags: ["Users"]
+      description: Returns the teachers for a user
+      operationId: getTeachersForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/TeachersResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/resources:
+    get:
+      tags: ["Users"]
+      description: Returns the resources for a user
+      operationId: getResourcesForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/ResourcesResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/schools:
+    get:
+      tags: ["Users"]
+      description: Returns the schools for a user
+      operationId: getSchoolsForUser
+      parameters:
+      - in: path
+        name: id
+        required: true
+        type: string
+      - in: query
+        name: limit
+        type: integer
+      - in: query
+        name: starting_after
+        type: string
+      - in: query
+        name: ending_before
+        type: string
+      responses:
+        '200':
+          description: OK Response
+          schema:
+            $ref: '#/definitions/SchoolsResponse'
+        '404':
+          $ref: '#/responses/NotFound'
+
+  /users/{id}/sections:
```

Comment:
Another minor callout: for this endpoint in AVS, we re-used existing code (getUserSections, getTeacherSections, getStudentSections have a lot of shared code) and it doesn't currently implement paging. I believe it's just for this endpoint, but you'd have to implement paging on top of the AVS call.

Reply:
Good callout, will keep that in mind